src/utils.ts

```diff
@@ -329,3 +330,25 @@ export const getFingerprint = (publicKey: Uint8Array): number => {
 
   return view.getUint32(0, false);
 };
+
+/**
+ * Get a secret recovery phrase (or mnemonic phrase) in string form as a
+ * `Uint8Array`. The secret recovery phrase is split into words, and each word
+ * is converted to a number using the BIP-39 word list. The numbers are then
+ * converted to bytes, and the bytes are concatenated into a single
+ * `Uint8Array`.
+ *
+ * @param mnemonicPhrase - The secret recovery phrase to convert.
+ * @returns The `Uint8Array` corresponding to the secret recovery phrase.
+ */
+export function mnemonicPhraseToBytes(mnemonicPhrase: string): Uint8Array {
+  const words = mnemonicPhrase.split(' ');
+  const indices = words.map((word) => {
+    const index = englishWordlist.indexOf(word);
+    assert(index !== -1, `Invalid mnemonic phrase: Unknown word "${word}".`);
+
+    return index;
+  });
+
+  return new Uint8Array(new Uint16Array(indices).buffer);
```

Comment:
Why is the `Uint8Array` created using a `Uint16Array`? 🤔

Reply:
The indices can be anything from 0 to 2047, which is too large to fit in an 8-bit number. By creating a `Uint16Array` and taking its underlying `ArrayBuffer`, we can easily split the 16-bit numbers into 8-bit numbers.

This is how it's done in `@metamask/scure-bip39` as well.

Reply:
Interesting 👍